src/components/view/pages/shoppingCart/cartPageListView.ts

```diff
@@ -0,0 +1,136 @@
+              <div class="cart-item__number">${cartItem.id}</div>
+              <img class="cart-item__img" data-id="${cartItem.product.id}" src="${cartItem.product.images[0]}" alt="product image">
+              <div class="cart-item__info">
+                <div class="cart-item-info__name">${cartItem.product.name}</div>
+                <div class="cart-item-info__color">Color: ${cartItem.product.color}</div>
+                <div class="cart-item-info__collecrion">Collection: ${cartItem.product.collection}</div>
+                <div class="cart-item-info__size">Size: ${cartItem.product.size}cm</div>
+                <div class="cart-item-info__category">Category: ${cartItem.product.category}</div>
+                <div class="cart-item-info__instock" data-id="${cartItem.product.id}">In stock: ${cartItem.product.stock}</div>
+              </div>
+             <div class="cart-item__price">$${cartItem.product.price}</div>
+             <div class="cart-item__qty">
+               <div class="cart-item-qty__value-container">
+                 <input class="cart-item-qty__value-container quantity-input" type="number" data-id="${cartItem.product.id}" data-stock="${cartItem.product.stock}" value="${cartItem.quantity}">
+               </div>
+               <div class="cart-item-qty__arrow-container arrow-up" data-stock="${cartItem.product.stock}" data-id="${cartItem.product.id}">
+                 <div class="cart-item-qty__arrow-up"></div>
+              </div>
+              <div class="cart-item-qty__arrow-container arrow-down" data-id="${cartItem.product.id}">
+                 <div class="cart-item-qty__arrow-down"></div>
+              </div>
+             </div>
+             <div class="cart-item__subtotal">$${formatPrice(cartItem.product.price * cartItem.quantity)}</div>
+            </div>
+            <div class="cart-item__cross" data-id="${cartItem.product.id}"></div>
+          </div>`
+        }).join('')
+    }
+
+    afterRender(controller: Controller) {
+      super.afterRender(controller);
+
+        document.querySelectorAll<HTMLElement>('.arrow-up').forEach((button: Element) => {
+          button.addEventListener('click', (event: Event) => {
+            const button = event.currentTarget as HTMLElement
+              controller.addProductToCart(Number(button.dataset.id));
+
+                // трести in stock если стрелкой вверх внесено большее количество чем есть
+                document.querySelectorAll<HTMLInputElement>(".quantity-input").forEach((input) => {
+                  if (Number(input.value) === (Number(button.dataset.stock))) {
+                      document.querySelectorAll<HTMLElement>(".cart-item-info__instock").forEach((stock: HTMLElement) => {
+                        if (Number(button.dataset.id) === Number(stock.dataset.id)) {
+                          stock.classList.add("shake-cart");
+                          setTimeout(() => stock.classList.remove("shake-cart"), 3000);
```

Comment:
You use this piece of code in several places, please try to use common func for it